ddr/lib/ddr-scanner/dwarf/DwarfParser.cpp

```diff
@@ -465,8 +465,8 @@ parseAttribute(char *line, Dwarf_Die *lastCreatedDie,
 					setError(error, DW_DLE_MAF);
 					newAttr->_udata = 0;
 				} else if (DW_TAG_unknown != (*lastCreatedDie)->_tag) {
-					Dwarf_CU_Context::_fileList.push_back(string(newAttr->_stringdata));
 					newAttr->_udata = Dwarf_CU_Context::_fileList.size();
+					Dwarf_CU_Context::_fileList.insert(make_pair(string(newAttr->_stringdata), newAttr->_udata));
```

Comment:
This is still not right:
* `newAttr->_udata` must be 1-based
* it should use an existing entry for the file name in `newAttr->_stringdata `